dispatcher/backend/requirements.txt

```diff
@@ -2,7 +2,8 @@ Flask>=2.3.2,<2.4
 flask-cors>=3.0,<3.1
 paramiko>=3.1,<3.2
 PyJWT>=2.6,<2.7
-pymongo==3.12.0
+# pymongo is still a requirement because used for bson serializer/deserializer
+pymongo==3.12.0 
```

Comment:
Then use `bson` directly

Reply:
If you speak about https://pypi.org/project/bson/, I have to warned you that this is not the official package maintained by Mongo. It is a fork by some folk http://github.com/py-bson/bson and looks not maintained at all.
I saw this package but decided to not use it due to the associated risks (no maintenance, potential risk of takeover by someone that would introduce a security flaw).
Are you aware of all this and do you still consider this is the way to go ?

Reply:
I forgot! Thanks for the reminder. Can you mention it in the comment here so that we don't accidentally replace it later ; it's uncommon that a single package installs multiple modules.

Also, there are traces of `ObjectId` in `utils/json.py` and `utils/token.py` that can be removed